nvi-commons/src/main/java/no/sikt/nva/nvi/common/db/NviCandidateRepository.java

```diff
@@ -105,6 +105,21 @@ public void saveNote(UUID candidateIdentifier, DbNote dbNote) {
         noteTable.putItem(newNoteDao(candidateIdentifier, dbNote));
     }
 
+    public Candidate updateCandidateAndRemoveApprovals(UUID identifier, DbCandidate dbCandidate,
```

Comment:
The name `updateCandidateAndRemoveApprovals` is misleading since updating a candidate entails removing approvals, so the name is just `updateCandidate`, or `updateCandidateRemovingApprovals`

Reply:
There is one difference between those two methods, updateCandidate() resets approvals, i.o.w. changing status to PENDING and removes assignees, while this method deletes all associated approvals from database. 
`approvalStatuses.forEach(approvalStatus -> transaction.addDeleteItem(approvalStatusTable, approvalStatus));`

Reply:
It think we want something in method name that says that we are actually removing approvals, what do you think? @brinxmat

Reply:
In which case, `updateCandidateRemovingApprovals`, I am obviously averse to the word `and`